psh/test-touch-rootfs.py

```diff
@@ -12,13 +12,14 @@
 # %LICENSE%
 
 import psh.tools.psh as psh
-from psh.tools.common import assert_mtime
+from psh.tools.common import assert_dir_mtimes
 
 
 def assert_bin(p):
     ''' Asserts that all files in /bin are able to be touched'''
     dates = {}
-    for file_info in psh.ls(p, 'bin'):
+    # Due to the long execution time on zynq-qemu target (especially during CI) after reboot timeout is increased
+    for file_info in psh.ls(p, 'bin', timeout=40):
```

Comment:
don't like increasing the timeout. Does it exceed quite often, or is it not a significant issue?

Reply:
quite often, from my test the times were:

- 30s
- 31s
- 27s
- 34s